lib/membrane_element_file/sink_multi.ex

```diff
@@ -38,15 +38,15 @@ defmodule Membrane.Element.File.Sink.Multi do
                 path/to/file0.ext, path/to/file1.ext, ...
                 """
               ],
-              split_event_type: [
-                type: :atom,
-                default: :split,
-                description: "Type of event causing switching to a new file"
+              split_event: [
+                type: :struct,
```

Comment:
I don't think its a good idea to pass struct instance here. It's safer to pass struct module name. Imagine a situation where we'd like to use non-empty struct as `split_event`. The options would have to contain some values for enforced keys and that's not something we want